static/app/components/discover/transactionsList.tsx

```diff
@@ -445,7 +447,18 @@ class TransactionsTable extends React.PureComponent<TableProps> {
             <GuideAnchor target="span_op_relative_breakdowns">
               <SortLink
                 align={align}
-                title={title}
+                title={
+                  title === t('operation duration') ? (
+                    <span>
+                      {title}
+                      <ExternalLink href="https://docs.sentry.io/product/performance/event-detail/#operations-breakdown">
```

Comment:
Hmm the pattern I notice everywhere else is that the StyledIconQuestion is used to indicate a tooltip, maybe there's a better icon to use or you could punt info about the ops breakdown in a tooltip instead?

Reply:
Not sure if it's a big deal though

Reply:
@jeremybourque @doralchan thoughts?

Reply:
We decided to take take out the link for now and just go with the tooltip on the ops breakdown change 😄